src/OnboardingSPA/styles/_branding.scss

```diff
@@ -27,6 +27,8 @@ body {
 		--nfd-onboarding-border: 219, 219, 219;
 		--nfd-onboarding-drawer-icon-fill: var(--nfd-onboarding-tertiary);
 		--nfd-onboarding-drawer-icon-active-fill: var(--nfd-onboarding-light);
+		--nfd-onboarding-loader-start: var(--nfd-onboarding-primary);
+		--nfd-onboarding-loader-end: var(--nfd-onboarding-tertiary);
```

Comment:
Should we be adding these to the Newfold CSS as well?

Reply:
I thought so too, but I do not see `--nfd-onboarding-tertiary`  and some other corresponding branding variables in `_interface.css` . We would need to get the respective data.

Reply:
Sounds good looks like we'll be doing that later on, rest all looks good to me. Approving, will do a QA and merge this.